### PR TITLE
[SDAN-264] - Changes to email templates

### PR DESCRIPTION
**Pull request by akintolga**

### CODE REVIEW

**Comment by petrjasek** [comment]
would be good to introduce a layout for emails so we have the footer consistent everywhere

**Comment by petrjasek** [comment]
would rather use a layout so we can add a common header as well if needed, like in https://github.com/superdesk/superdesk-core/blob/master/superdesk/templates/email_layout.html

File: newsroom/templates/email_layout.html [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<p>{{ name }},</p>
+
+{% block content %}{% endblock %}
+
+{% include "email_footer.html" %}
```
Comment (petrjasek) [inline newsroom/templates/email_layout.html]: now you can add the footer here instead of including it, or is it included anywhere else?

Comment (akintolga) [reply newsroom/templates/email_layout.html]: 🥇 <-- This is what I need I guess

**Pull request merged**